spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -313,16 +313,19 @@ def main(argv=None):
         print((status, output), '\n')
 
     # check PropSeg compatibility with OS
-    print_line('Check PropSeg compatibility with OS ')
-    status, output = run_proc('isct_propseg', verbose=0, raise_exception=False, is_sct_binary=True)
-    if status in (0, 1):
-        print_ok()
+    if sys.platform.startswith('win32'):
+        pass  # sct_propseg is not yet supported on Windows (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3694)
```

Comment:
Is it possible we might want to output some message anyway? e.g.

```python
print_line("Skipping PropSeg compatibility check (sct_propseg is not yet supported on Windows)")
```

It's a little more transparent, but it also might be bad optics to inform users of a feature they _don't_ have access to before they try to use it. 🤔 

I don't have strong feelings either way, but if there's a specific use case where one or the other would be important, I could definitely be convinced!

Reply:
I've slightly tweaked the message just to make it blend in a little better, formatting wise:

![image](https://user-images.githubusercontent.com/16181459/161813507-341ea880-1af2-48bd-a4c9-014582a991d8.png)

Reply:
The wording for this message came up in our SCT meeting, and it was suggested that simply saying "Windows" is a bit vague, since we have two different types of Windows installation methods (native and virtualization (i.e. WSL/docker/etc.)).

So, Julien proposed the following wording:

> “Skipping PropSeg compatibility check (sct_propseg is not yet supported on Windows ‘Native’ installation (ie: without WSL))")

I think alluding to WSL would be an improvement, since it could guide users to the WSL-based installation of Windows if they truly need propseg over deepseg.

(Though, I'm not sure we want to edit the proposed message to be a bit more brief, just for horizontal space concerns.

Reply:
--> [b44d97d](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3753/commits/b44d97dcd676d0795bc0519917c9e52b65c9f2c6) 😄